### ISSUE TITLE
Label conditions tab not displaying properly

### ISSUE THREAD

**Issue by Yotin1** [DelvUI/DelvCD#4]
Sometimes, when I open the label conditions tab, it will look like this:

![image](https://github.com/DelvUI/DelvCD/assets/58013144/58283881-158f-49a4-b25c-b24ce0a405c8)

~~It seems that switching to the job the UI elements are visible for fixes it.~~
Making sure the element the label is for is visible seems to fix it. However, this bug only occurs for some elements that aren't visible.

**Comment by ShepardToTheStars** [comment]
I actually ran into this myself as well. I'm looking into it. For posterity and reference, this is the error I was getting from the logs.

```
2023-10-29 13:24:06.758 -05:00 [ERR] [WindowSystem] Error during Draw(): DelvCD  >  Group [AST]  >  Group [Reminders]  >  Icon [DOT BADDIES - Combust III]  >  Label [Countdown]
System.ArgumentException: '0' cannot be greater than -1.
   at System.Math.ThrowMinMaxException[T](T min, T max)
   at DelvCD.Config.StyleConditions`1.DrawStyleConditionRow(Int32 i) in C:\Users\windowsuser\src\DelvCD\DelvCD\Config\Styles\StyleConditions.cs:line 249
   at DelvCD.Config.StyleConditions`1.DrawConfig(IConfigurable parent, Vector2 size, Single padX, Single padY) in C:\Users\windowsuser\src\DelvCD\DelvCD\Config\Styles\StyleConditions.cs:line 202
   at DelvCD.Windows.ConfigWindow.Draw() in C:\Users\windowsuser\src\DelvCD\DelvCD\Windows\ConfigWindow.cs:line 97
   at Dalamud.Interface.Windowing.Window.DrawInternal(DalamudConfiguration configuration) in C:\goatsoft\companysecrets\dalamud\Interface\Windowing\Window.cs:line 329
```

**Comment by Tischel** [comment]
Fixed in v0.5.2.0 (I think)
